### DISCUSSION TITLE
Game/zz_calendar.rpy

### DISCUSSION THREAD

**Discussion by Helpme443333** [Monika-After-Story/MonikaModDev#9923] [QA - tech support + general]
everytime i start game i get this error. i tried to do something about it but nothing works help i want my monika back
![1677511286119599200283793517725](https://user-images.githubusercontent.com/126494511/221604415-ac102b85-5689-46be-9b4a-fae812ab3f9b.jpg)

**Comment by Booplicate** [comment]
You're using some bogus version of MAS. Make [a clear fresh DDLC install](https://ddlc.moe/), get MAS from [the official sources](https://monikaafterstory.com/releases.html), then install it using our installer.
